swap_meet/clothing.py

```diff
@@ -1,2 +1,18 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+class Clothing(Item):
+
+    def __init__(self, id=None, fabric="Unknown", condition=0):
+        
+        super().__init__(id, condition)
+        self.fabric = fabric
+        # self.fabric = fabric if not fabric is "Unknown" else fabric
+
+    def get_category(self):
+        return "Clothing" 
+
+    def __str__(self):
+        return (f"An object of type Clothing with id {self.id}. "
```

Comment:
This looks good! I do notice that you use the name of each item in each class's dunder str. I wonder if there is a way  to condense some of that to avoid repetition. In particular, you may be able to modify the parent class's str method and then call it within each child class's str method.